lms/templates/certificates/accomplishment-base.html

```diff
@@ -9,7 +9,7 @@
 %>
 
 <!DOCTYPE html>
-<html class="no-js" lang="en">
```

Comment:
Removing this does not fix the issue. You need to set it dynamically to whatever the current language is, as otherwise screenreaders will not be able to properly read the translated strings.

Reply:
Sanity Check: Is the LANGUAGE_CODE variable here dynamically set according to the current active language for the page, or is it the application default language?

Reply:
@amangano-edx I can't able to find the place from where and how this LANGUAGE_CODE variable is injected into template. Cant able to find any such variable in context. I got idea and help from https://github.com/edx/edx-platform/blob/master/lms/templates/main.html#L27. Manually testing it shows the variable is correctly set for any active language when i change the user language via  http://localhost:18000/account/settings 
![lang_en](https://user-images.githubusercontent.com/373677/34724858-1b29741a-f571-11e7-80e5-4fa2760d13b8.png)
![lang_french](https://user-images.githubusercontent.com/373677/34724879-2233d2f0-f571-11e7-8ff6-5826d343f268.png)
![lang_arabic](https://user-images.githubusercontent.com/373677/34724880-2568f9dc-f571-11e7-8210-26e6e88f7567.png)

Reply:
It might be safer to use `get_language` (https://docs.djangoproject.com/en/2.0/ref/utils/#django.utils.translation.get_language), and only fallback to LANGUAGE_CODE when `get_language` returns `None`. The Django docs make it clear what `get_language` does, so I think it'd be be better to rely on that than on the observed behavior of `LANGUAGE_CODE`.

Reply:
Sorry, I think I misread your original comment. I think that if the main lms template uses LANGUAGE_CODE, then it is probably a good idea to use that here as well. Sorry for the confusion.